Dikke Ploaten/Controller/LaunchViewController.swift

```diff
@@ -23,13 +24,14 @@ class LaunchViewController : UIViewController {
         super.viewDidAppear(animated)
         animate(imageView: imgVinyl, images: vinylImages)
 		
-		// TODO: check
-//		if Auth.auth().currentUser != nil {
-//			self.performSegue(withIdentifier: "alreadyLoggedIn", sender: nil)
-//		}
+        //TODO: question about launscreen.storyboard
+        Database().checkUser {
+                // Go to mainview
+//                self.performSegue(withIdentifier: "alreadyLoggedIn", sender: nil)
```

Comment:
Hier ga je niet naar de ingelogde view? 🤔  
Dus een gebruiker die ingelogd is krijgt gewoon het launchscreen te zien en geraakt niet verder?